src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -59,17 +59,19 @@ export interface SensitivityProps {
   value?: SensitivityType;
   loading?: boolean;
   className?: string;
+  sx?: SxProps<Theme>;
```

Comment:
I've got a shortcut for this here
https://github.com/SeedCompany/cord-field/blob/2f3b79721181e77a674a7ffa1d741d9edf7f51fb/src/common/sx.ts#L5-L8